### PR TITLE
Post release changes for 1.3.0

### PR DESCRIPTION
**Pull request by sbethur**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.

~- [ ] I've added tests applicable for this pull request~

### What does this Pull Request accomplish?

Get the repo ready for new development after [1.3.0](https://github.com/ni/nimi-python/releases/tag/1.3.0) release

### List issues fixed by this Pull Request below, if any.

None

### What testing has been done?

Visual inspection of generated files. CI will run all tests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +23,49 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased
+* ### ALL
+    * #### Added
+    * #### Changed
+    * #### Removed
+* ### NI-DCPower
```
Comment (marcoskirsch) [inline CHANGELOG.md:31]: Pre-existing, but I think we should use the module name here. Something like:

```
* ### `nidcpower` (NI-DCPower)
```

Can we make this change in a separate PR?

File: docs/nidigital/examples.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Examples
 ========
 
-`You can download all nidigital examples here <https://github.com/ni/nimi-python/releases/download/1.3.0/nidigital_examples.zip>`_
+`You can download all nidigital examples for latest version here <https://github.com/ni/nimi-python/releases/download/1.3.0/nidigital_examples.zip>`_
```
Comment (marcoskirsch) [inline docs/nidigital/examples.rst:4]: This is wrong!

Comment (marcoskirsch) [reply docs/nidigital/examples.rst:4]: ok not wrong just weird...

Comment (marcoskirsch) [reply docs/nidigital/examples.rst:4]: Link says 1.3.0 yet there is no 1.3.0 of nidigital.

File: CHANGELOG.md (hunk 1 above)
Comment (sbethur) [reply CHANGELOG.md:31]: Agreed.

Changed in https://github.com/ni/nimi-python/pull/1470.
